### DISCUSSION TITLE
To use the ENVs to define mediamtx.yml in Docker context

### DISCUSSION THREAD

**Discussion by AlanTuring1988** [bluenviron/mediamtx#2753] [Questions]
### Question

Hi,

Can I use the ENVs to define the auth sections on mediamtx.yml?

In docker context , I would like to use the ENVs to have a mediamtx.yml configuration dynamically. I tried to set the yaml file using the placeholder ${env_name} for publishUser  and publishPass but when I try to connect via rmtp protocol I had an auth error.

Tnks,
Pietro.

**Comment by aler9** [comment]
Hello, all configuration parameters can be overridden by using environment variables, including the auth section, as described in the README:
https://github.com/bluenviron/mediamtx#configuration

If you still have problems, please report the server version, the OS, the command line you used to launch the Docker container and the full server log.

[a comment by an automated account is not shown]
